test/integration/transaction_create_edit_test.rb

```diff
@@ -75,7 +75,7 @@ class TransactionCreateEditTest < JavascriptIntegrationTest
       visit_edition transaction
 
       fill_in "Service analytics profile", with: "UA-INVALID-SPACE-FLIGHT"
-      save_edition_and_assert_error
```

Comment:
As per suggestion earlier I think it's good to keep the `and_assert_error` suffix, it seems a logical counter to the next assertion of `save_edition_and_assert_success`. It'd be a good idea to also allow an error to be passed in which could be asserted against too (e.g. "can't be blank")

Reply:
Updated as per your suggestion.